### DISCUSSION TITLE
laravel parallel test cant create the database

### DISCUSSION THREAD

**Discussion by farshadff** [paratestphp/paratest#816] [Q&A]
i am trying to run laravel feature tests in parallel mode as it is in document . my `phpunit.xml` is configed as below :
```
    <php>
        <env name="APP_ENV" value="testing"/>
        <env name="BCRYPT_ROUNDS" value="4"/>
        <env name="CACHE_DRIVER" value="array"/>
        <env name="SESSION_DRIVER" value="array"/>
        <env name="QUEUE_DRIVER" value="sync"/>
        <env name="MAIL_DRIVER" value="array"/>
        <env name="DB_CONNECTION" value="mysql"/>
    </php>
```
and i am running the tests with this command :
```
php artisan test --parallel
```
the result is all tests fail with this error message :
```
PDOException: SQLSTATE[HY000] [1049] Unknown database 'mydbname_test_11'
```
should i create this database before starting test or laravel creates that automatically ?

**Comment by Slamdunk** [comment]
I have no idea, this repo knows nothing about Laravel, but I can give you this link https://laravel.com/docs/8.x/testing#running-tests-in-parallel
